src/inmanta/module.py

```diff
@@ -1486,9 +1486,16 @@ def to_config(self, inp: Optional[configparser.ConfigParser] = None) -> configpa
 
 
 @stable_api
-class ModuleRepoType(enum.Enum):
+class ModuleRepoType(DeprecatedEnum):
```

Comment:
Wouldn't a pydantic validator be way simpler?

Reply:
Indeed it is, I removed all of it and when with a validator :+1: